src/util.rs

```diff
@@ -0,0 +1,82 @@
+use merkle::proof::{Lemma, Positioned, Proof};
```

Comment:
Can you give a more descriptive name to the `util` module?

Reply:
Good point! Maybe `fmt`, since it now contains only formatting? I renamed it, but I'm open to better suggestions.